### ISSUE TITLE
Github logo link is missing aria label

### ISSUE THREAD

**Issue by emmalearnscode** [EddieHubCommunity/LinkFree#648]
### Description

The github logo in the search bar is a link but as it doesn't have any text it needs an aria-label to specify for screen readers where the link leads.

### Screenshots

<img width="295" alt="Screenshot 2021-11-09 142758" src="https://user-images.githubusercontent.com/57045550/140934106-1cb43153-0638-4361-bc12-fbfbc614bc36.png">

**Comment by krishguptadev** [comment]
aria-label="View source on GitHub"
